src/modules/apps/e2e/errorBoundary.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { Color } from 'ink'
+
+export class ErrorBoundary extends React.Component<{}, { error: any }> {
+    constructor(props) {
+      super(props)
+      this.state = { error: null }
+    }
+  
+    static getDerivedStateFromError(error) {
+      // Update state so the next render will show the fallback UI.
+      return { error }
+    }
```

Comment:
Not sure what this does, it's not used anywhere and is static, so it's not related to an instance, maybe remove?